lmfit/confidence.py

```diff
@@ -388,8 +392,9 @@ def conf_interval2d(minimizer, result, x_name, y_name, nx=10, ny=10,
     y : numpy.ndarray
         Y-coordinates (same shape as `ny`).
     grid : numpy.ndarray
-        Grid containing the calculated probabilities (with shape
-        ``(nx, ny)``).
+        2-D array ((with shape ``(nx, ny)``) containing the calculated
+        probabilities
+        or chi-square
```

Comment:
this probably shouldn't be on a separate line?

Reply:
will fix.